compiler/src/iree/compiler/DispatchCreation/FormDispatchRegions.cpp

```diff
@@ -145,6 +147,49 @@ class FusionGroup {
   // Insert `op` into the fusion group.
   void insert(Operation *op);
 
+  /// Returns true if `consumerOp` has a transitive dependency on the fusion
+  /// group. This means that some transitive dependency of `consumerOp` (not in
+  /// the fusion group) itself uses an operation in the fusion group. This is
+  /// required for fusion because it must be legal to take a program slice that
+  /// contains only the ops in the fusion group.
+  bool hasTransitiveDependencyOnFusionGroup(Operation *consumerOp) const {
+    BackwardSliceOptions options;
+    DominanceInfo dominance(consumerOp);
```

Comment:
Does the Dominator need to be put in here? I prefer to pass the dominator in as an operand.